test/smoke/src/debugIos.test.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import { SpectronApplication } from "./spectron/application";
+import * as assert from "assert";
+import { AppiumHelper, Platform } from "./helpers/appiumHelper";
+import { SmokeTestsConstants } from "./helpers/smokeTestsConstants";
+import { RNworkspacePath } from "./main";
+import { IosSimulatorHelper } from "./helpers/iosSimulatorHelper";
+import { Client, RawResult } from "webdriverio";
+import { sleep, runInParallel } from "./helpers/utilities";
+import { SetupEnvironmentHelper } from "./helpers/setupEnvironmentHelper";
+
+const RN_APP_BUNDLE_ID = "org.reactjs.native.example.latestRNApp";
+const RNDebugConfigName = "Debug iOS";
+// const ExpoDebugConfigName = "Debug in Exponent";
+// Time for OS Debug Test before it reaches timeout
+const debugIosTestTime = SmokeTestsConstants.iosAppBuildAndInstallTimeout + 100 * 1000;
+// Time for iOS Expo Debug Test before it reaches timeout
+// const debugExpoTestTime = smokeTestsConstants.expoAppBuildAndInstallTimeout + 400 * 1000;
+export function setup() {
+    describe("Debugging iOS", () => {
+        let clientInited: Client<RawResult<null>> & RawResult<null>;
+        before(async function () {
+            const app = this.app as SpectronApplication;
```

Comment:
This code is duplicated here and [here](https://github.com/Microsoft/vscode-react-native/pull/988/files#diff-8be38092440890d75c628f5887a2b044R38)

Reply:
@SounD120 we have to do that in order to get app as SpectronApplication and do stuff with it.